.travis.yml

```diff
@@ -2,9 +2,10 @@
   "language": "nix",
   "env": {
     "jobs":
-      [ "GHC_VERSION=ghc844"
-      , "GHC_VERSION=ghc865"
-      , "GHC_VERSION=ghc883"
+      [ "GHC_VERSION=ghc8104"
+      , "GHC_VERSION=ghc884"
+      , "GHC_VERSION=ghc901"
+      , "GHC_VERSION=ghcHEAD"
```

Comment:
Thanks for bumping! Do the older versions (8.6 and 8.4) not work anymore with this change? Or can we keep them?

Reply:
The older versions don't exist in the pinned nixpkgs version. We can add them if necessary...  but I'd have to do some research/investigation for how to support that.